pom.xml

```diff
@@ -64,6 +72,24 @@
         <version>1.3</version>
       </dependency>
       <dependency>
+        <groupId>org.jenkins-ci.plugins</groupId>
+        <artifactId>junit</artifactId>
+        <version>1.10</version>
+        <scope>test</scope>
+      </dependency>
+      <dependency>
+        <groupId>org.jenkins-ci.plugins</groupId>
+        <artifactId>credentials</artifactId>
+        <version>1.24</version>
+        <scope>test</scope>
+      </dependency>
```

Comment:
(Note for me)
credentials-plugin is not necessary to have tests succeeds.
Though, it is useful to suppress error messages from folder-plugin.